vmware-event-router/internal/processor/aws_event_bridge.go

```diff
@@ -176,8 +177,9 @@ func NewAWSEventBridgeProcessor(ctx context.Context, cfg connection.Config, sour
 func (awsEventBridge *awsEventBridgeProcessor) Process(moref types.ManagedObjectReference, baseEvent []types.BaseEvent) error {
 	batchInput, err := awsEventBridge.createPutEventsInput(baseEvent)
 	if err != nil {
-		awsEventBridge.Printf("could not create PutEventsInput for event(s): %v", err)
-		return nil
+		errMsg := fmt.Errorf("could not create PutEventsInput for event(s): %v", err)
+		awsEventBridge.Printf(errMsg.Error())
```

Comment:
```suggestion
		awsEventBridge.Println(errMsg)
```

Reply:
^ is that suggestion ok?

Reply:
Do we need to change lines in OpenFaaS proc?

Reply:
Changed to `awsEventBridge.Println(errMsg)` ✔️  The openfaas error comes in the struct not through Invoke method, so we should think how exactly those errors should be added to the OF processors